src/UI/examples/Item/Contribution/base.php

```diff
@@ -0,0 +1,16 @@
+<?php declare(strict_types=1);
+
+namespace ILIAS\UI\examples\Item\Contribution;
+
+function base()
```

Comment:
Why do you use a function here instead of a class? This makes extending/overwriting it very very hard. This applies to all the other functions.

Reply:
@PurHur This is an example, this code MUST not be used in productive code, nor SHOULD it be derived.

Reply:
Wouldnt a example be better if it shows how to use in a real code environment? As outside developer i have no clue what todo with that function.

Reply:
Why would you want to extend the example? What is a real "code environment"? 

The idea here is to show a few short examples of how the component can be generated and rendered, next to the actual output that is rendered by the code. For me, this is rather useful. It does not matter if I would wrap a class around it. If I would use the code, I would but it again into a function (probably inside a class or trait), not?

Further it is handy for testers, to have one central place where they can check if each UI Component looks like it is supposed to.

Reply:
@PurHur Thx for the review!
As said these functions arent for production or to use inside actual content structure, but to give developers exaples about the usage of new components. There is no usecase for that, its more like a documentation